packages/api-bindings/src/lens/__helpers__/mutations/profile.ts

```diff
@@ -86,3 +79,170 @@ export function mockCreateSetProfileMetadataTypedDataResponse({
     result: { data },
   };
 }
+
+export function mockCreateFollowTypedDataData({
+  nonce = mockNonce(),
+}: { nonce?: Nonce } = {}): CreateFollowTypedDataData {
+  return {
+    result: mockCreateTypedDataResult('CreateFollowBroadcastItemResult', {
+      types: {
+        Follow: [mockEIP712TypedDataField()],
+      },
+      domain: mockEIP712TypedDataDomain(),
+      message: {
+        nonce,
+        deadline: 1644303500,
+        followerProfileId: mockProfileId(),
+        idsOfProfilesToFollow: [mockProfileId()],
+        followTokenIds: [],
+        datas: [],
+      },
+    }),
+  };
+}
+
+export function mockCreateFollowTypedDataResponse<T extends CreateFollowTypedDataData>({
+  variables,
+  data,
+}: {
+  variables: CreateFollowTypedDataVariables;
+  data: T;
+}): MockedResponse<T> {
+  return {
+    request: {
+      query: CreateFollowTypedDataDocument,
```

Comment:
I really wished we used `TypeDocumentNode`... `CreateFollowTypedDataData` and `CreateFollowTypedDataVariables` could be inferred from that.

Reply:
it shouldn't be a big deal to rerun codegen with TypeDocumentNode setting, maybe something to consider for later